src/components/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -25,19 +26,28 @@ const Template: ComponentStory<typeof ButtonGroup> = (args) => (
   <ButtonGroup {...args} />
 );
 
+export const WithState = () => {
+  const [value, setValue] = useState('click', 'Ukraine');
```

Comment:
I am not completely sure, but as much as I understood we need to assign what type of action is going to be. If it's button click, then as example `useState('click', '')`, but honestly I tried to write various things and it did not seem to me like it does something 😄 but we need to write something or it will give an error.

Reply:
The problem with storybook-addon-state is that it seemingly [hasn't been updated for 3 years](https://storybook.js.org/addons/storybook-addon-state).
I think I've found another way to do it using the [useArgs hook](https://storybook.js.org/docs/react/addons/addons-api#useargs) from the Storybook API following [this post](https://medium.com/urbandataanalytics/changing-args-values-state-in-storybook-without-usestate-hook-15cc096716f7). I've pushed examples in ButtonGroup.stories.tsx and SearchBar.stories.tsx using that, can you have a look if it works for the others as well?

Reply:
That's really good catch, but I actually do not know where my brain was at, but now noticed this https://storybook.js.org/docs/react/writing-stories/introduction#working-with-react-hooks. Looks like we can use `useState` straight from React and we do not need anything additional. This seems to be working for me. Just let me know what you are more into using and will add changes to this PR.

Reply:
Oh right, even better, just use React hooks then 👍